source/faq.rst

```diff
@@ -52,10 +53,60 @@ How do I navigate to another Pod?
 To browse another Pod (or even a different Container in your own Pod), you can navigate there 
 directly by entering the Pod (or Container) URI.
 
-#. In PodBrowser, click the current :guilabel:`Pod` identifier link (just below the application logo).
+#. In PodBrowser, click the current :guilabel:`Pod` identifier link (just below the Account menu).
+
+#. Select :guilabel:`Change Pod` in the menu.
+
+#. In the displayed panel, enter the URI for the Pod (or Container).
+
+#. Press :guilabel:`Enter`, or click the :guilabel:`Go` button.
+
+#. The navigation panel is refreshed and shows the Resources in the entered Pod (or Container).
+
+What do the Resource permissions mean?
+======================================
+
+PodBrowser allows you to share Containers and Resources within your
+Pod with other Solid users. But what do the available permissions allow the other Solid users to do? 
+The table below describes the permissions which can be conbined together:
+
+.. list-table::
+  :header-rows: 1
+  :widths: 20 80
+
+  * - Permission
+
+    - Other Solid user can...
+
+  * - View
+
+    - View the Resources within a Container, or download a Resource.
+
+  * - Edit
+
+    - Change the contents of or delete a Resource.
+
+  * - Append
+
+    - Add to a Resource, but not change the existing contents.
+
+  * - Control
+
+    - Change the permissions for a Resource.
+      
+.. note:: To allow other Solid users to display the Resource details in PodBrowser, 
```

Comment:
To allow other Solid users to display the Resource details (including the Download and Delete options) in PodBrowser, the users must have View access to the Resource.